mlflow/gateway/schemas/chat.py

```diff
@@ -24,72 +22,56 @@ class RequestPayload(BaseRequestPayload):
 
     class Config:
         schema_extra = {
-            "example": {
-                "messages": [
-                    {
-                        "role": "user",
-                        "content": "hello world",
-                    }
-                ],
-                "temperature": 0.0,
-                "max_tokens": 64,
-                "stop": ["END"],
-                "candidate_count": 1,
-            }
+            "messages": [
+                {"role": "system", "content": "You are a helpful assistant."},
+                {"role": "user", "content": "Hello!"},
+            ],
+            "temperature": 0.0,
+            "max_tokens": 64,
+            "stop": ["END"],
+            "n": 1,
         }
 
 
-class FinishReason(str, Enum):
-    STOP = "stop"
-    LENGTH = "length"
-
-
 class ResponseMessage(ResponseModel):
     role: str
     content: str
 
 
-class CandidateMetadata(ResponseModel, extra="allow"):
-    finish_reason: Optional[FinishReason] = None
-
-
-class Candidate(ResponseModel):
+class Choice(ResponseModel):
+    index: int
     message: ResponseMessage
-    metadata: CandidateMetadata
+    finish_reason: Optional[str] = None
 
 
-class Metadata(ResponseModel):
-    input_tokens: Optional[int] = None
-    output_tokens: Optional[int] = None
+class ChatUsage(ResponseModel):
+    prompt_tokens: Optional[int] = None
+    completion_tokens: Optional[int] = None
     total_tokens: Optional[int] = None
-    model: str
-    route_type: RouteType
 
 
 class ResponsePayload(ResponseModel):
-    candidates: List[Candidate]
-    metadata: Metadata
+    id: Optional[str] = None
+    object: str = "chat.completion"
```

Comment:
```suggestion
    object: Literal["chat.completion"]
```

Reply:
`Literal` can be imported from `typing`.

Reply:
https://github.com/mlflow/mlflow/pull/10457 for embeddings.

Reply:
Love it - thanks!